test/hooks/useSelector.spec.tsx

```diff
@@ -380,20 +406,30 @@ describe('React', () => {
           const spy = jest.spyOn(console, 'error').mockImplementation(() => {})
 
           const Parent = () => {
-            const count = useSelector((s) => s.count)
+            const count = useSelector<DefaultRootState, number>((s) => s.count)
```

Comment:
:hand: Similarly, here the return type should be inferred correctly if the state is typed right